tendrl/node_agent/manager/tendrl_definitions_node_agent.py

```diff
@@ -137,6 +137,16 @@
           run: tendrl.node_agent.atoms.service.configure.Configure
           type: Update
           uuid: b90a0d97-8c9f-4ab1-8f64-dbb5638159a3
+        check_service_status:
+          enabled: true
+          inputs:
+            mandatory:
+              - Node.fqdn
+              - Service.name
+          name: "check whether the service is running"
+          run: tendrl.node_agent.objects.service.atoms.check_service_status.CheckServiceStatus
+          type: Create
```

Comment:
Atom names can be re-verified in future, type is not really used much. we can move on